1-exercises/A-array-find/exercise.js

```diff
@@ -17,8 +17,9 @@ let names = [
   "Ahmed",
 ];
 
-let longNameThatStartsWithA = findLongNameThatStartsWithA(names);
 
+let longNameThatStartsWithA =  names.find((element) => element[0] === "A" && element.length > 7);
```

Comment:
Technically correct. You could consider using the variable "name" instead of the variable "element".